docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -2,27 +2,26 @@
 
 = Migrating Ceph Monitor daemons to {Ceph} nodes
 
-The following section describes how to move Ceph Monitor daemons from the
-{rhos_prev_long} Controller nodes to a set of target nodes. Target nodes might
-be pre-existing {Ceph} nodes, or {OpenStackShort} Compute nodes if {Ceph} is
-deployed by {OpenStackPreviousInstaller} with an HCI topology.
-This procedure assumes that some of the steps are run on the source node that
-we want to decommission, while other steps are run on the target node that is
-supposed to host the redeployed daemon.
+You must move Ceph Monitor daemons from the {rhos_prev_long} ({OpenStackShort}) Controller nodes to a set of target nodes. Target nodes are either existing {Ceph} nodes, or {OpenStackShort} Compute nodes if {Ceph} is
+deployed by {OpenStackPreviousInstaller} with a Hyperconverged Infrastructure (HCI) topology.
 
+To migrate the Ceph Monitor daemons, the following actions occur:
 
-.Prerequisites
+* The mon IP addresses are moved to the target {Ceph} nodes. The IP address migration assumes that the target nodes are originally deployed by {OpenStackPreviousInstaller} and that the network configuration is managed by `os-net-config`.
+* The existing Controller nodes are drained and decommisioned.
+* Additional monitors are deployed to the target nodes, and they are promoted
+as `_admin` nodes that you can use to manage the {CephCluster} cluster and
+perform day 2 operations.
 
-Configure the target nodes (CephStorage or ComputeHCI) to have both `storage`
-and `storage_mgmt` networks to ensure that you can use both {Ceph} public and
-cluster networks from the same node. This step requires you to interact with
-{OpenStackPreviousInstaller}. From {rhos_prev_long} {rhos_prev_ver} and later
-you do not have to run a stack update. However, there are commands that you
-must perform to run `os-net-config` on the bare metal node and configure
-additional networks.
+.Prerequisites
 
-. If target nodes are `CephStorage`, ensure that the network is defined in the
-`metalsmith.yaml` for the CephStorageNodes:
+* The target nodes, CephStorage or ComputeHCI, are configured to have both `storage` and `storage_mgmt` networks. This ensures that you can use both {Ceph} public and cluster networks from the same node. This step requires you to interact with {OpenStackPreviousInstaller}. From {OpenStackShort} {rhos_prev_ver} and later you do not have to run a stack update.
++
+[NOTE]
+This step requires you to interact with {OpenStackPreviousInstaller}. From {OpenStackShort} {rhos_prev_ver} and later you do not have to run a stack update.
+* Run `os-net-config` on the bare metal node and configure additional networks:
```

Comment:
This sentence used to say, "However, there are commands that you must perform to run `os-net-config` on the bare metal node and configure additional networks.” I assumed that this sentence was referring to the provisioning steps in the prerequisites, so I changed it to "Run `os-net-config` on the bare metal node and configure additional networks:" to introduce the steps. Please confirm whether this is accurate.

Reply:
No, the original meaning of this was: "Usually you have to interact with tripleo to propagate networks to CephStorage nodes or ComputeHCI. However, if you're executing this procedure after OpenStack has been adopted, you might need to use `os-net-config` on the baremetal nodes to provision additional networks".
Now, the more I read this prerequisite step about networks, the more I'm wondering if we should move (in a follow up patch) this TripleO step at the very beginning of the adoption guide.
It would be something like: 
```
"If you have director deployed ceph, before starting the adoption procedure, complete the following steps: 
a. network propagation (which refers to this prereq section here)
b. if manila is part of the deployment and is backed by Ceph-NFS, build the NFS cluster
... (etc)

If ceph is not director deploy, but external:
a. if you have external RGW complete these steps
b. if you have manila with ceph-nfs managed by TripleO, complete these steps
"
```
I'm curious to hear your opinion about that. The goal is to performs some tasks using TripleO while the undercloud is still available, because if you follow the entire procedure and reach this point, undercloud is not there anymore (hence you can't run `openstack overcloud node provision ` ..), all the overcloud nodes have been adopted, and customers need to "manually" add the vlan to their CephStorage nodes, because there's no tool managing the network at that point.